### PR TITLE
Fixed the UBIFS file system extraction on python 3 environment

### PR DESCRIPTION
**Pull request by 0xd3xt3r**
Synced up with the latest version of the ubi_reader python package which is already ported to python 3 earlier version only worked in python 2

Related Issue: #425

### CODE REVIEW

**Comment by 0xd3xt3r** [comment]
@devttys0 any update ??

File: deps.sh [hunk 1]
```diff
@@ -118,11 +118,12 @@ function install_cramfstools
 
 function install_ubireader
 {
+    install_pip_package python-lzo
     git clone https://github.com/jrspruitt/ubi_reader
     # Some UBIFS extraction breaks after this commit, due to "Added fatal error check if UBI block extends beyond file size"
     # (see this commit: https://github.com/jrspruitt/ubi_reader/commit/af678a5234dc891e8721ec985b1a6e74c77620b6)
```
Comment (XVilka) [inline deps.sh]: Maybe remove this comment too?

Comment (0xd3xt3r) [reply deps.sh]: remove why ?

**Comment by 0xd3xt3r** [comment]
any update ?

File: deps.sh (hunk 1 above)
Comment (jameshilliard) [reply deps.sh:124]: The bug in the comment should be fixed [here](https://github.com/jrspruitt/ubi_reader/commit/63105bf3c0b1618f029da65d73e0ed7a0874e695).

**Pull request closed without merging**